tests/storages_tests/rdb_tests/create_db.py

```diff
@@ -6,13 +6,16 @@
 
 
 def objective_test_upgrade(trial: optuna.trial.Trial) -> float:
-    x = trial.suggest_float("x", -5, 5)  # optuna==0.9.0 does not have suggest_float.
-    y = trial.suggest_int("y", 0, 10)
+    x1 = trial.suggest_float("x1", -5, 5)  # optuna==0.9.0 does not have suggest_float.
+    x2 = trial.suggest_float("x2", -5, 5, step=2)
+    x3 = trial.suggest_float("x3", 1e-5, 1e-5, log=True)
+    y1 = trial.suggest_int("y1", 0, 10)
+    y2 = trial.suggest_int("y2", 1, 10, log=True)
     z = cast(float, trial.suggest_categorical("z", [-5, 0, 5]))
     trial.set_system_attr("a", 0)
     trial.set_user_attr("b", 1)
     trial.report(0.5, step=0)
-    return x ** 2 + y ** 2 + z ** 2
+    return x1 ** 2 + y1 ** 2 + z ** 2 + x2 + x3 + y2
```

Comment:
This change affects tests of older versions of storage. I'm wondering which approach to use: (a) entirely rewriting tests with updating test assets, and (b) create a new script (e.g. create_db_v3.py) and tests focusing on distribution migration.

Reply:
What do you think about it @toshihikoyanase?

Reply:
I'm very sorry for the delayed response. The current code that adds a new objective function for recent versions of Optuna seems reasonable to me.